Fix babysteps and baudrate

photodude opened the pull request:
### Description

This patch corrects two issues which cause significant issues in setup and tuning of a BIBO printer.

1. enables babysteps which are needed for bltouch z-position corrections
2. baudrate was corrected to 115200

### Benefits

fixes inability to adjust z position when using bltouch as babysteps were not enabled
Fixes issues where M114 would not report the printer position due to baudrate being above 115200

Note: Users with factory mks tf28 screens will need to modify the screen baudrate to 115200. The factory baudrate of 250000 causes problems with gcode items which report back information like M114. Using 115200 on both the mainboard and the screen resolves this issue.

photodude committed:
Activate babysteps

babysteps are needed for bltouch

photodude committed:
baudrate correction to 115200

factory default was 250000 but this causes issues on the stock MKS Gen L mainboard. random inconsistencies like M114 would not return the printer position. correcting the mainboard and the msktf28 screen to 115200 resolves this issue.

thisiskeithb commented:
> Note: Users with factory mks tf28 screens will need to modify the screen baudrate to 115200. The factory baudrate of 250000 causes problems with gcode items which report back information like M114. Using 115200 on both the mainboard and the screen resolves this issue.

Please add this to a `README.md` file and include it with the configs since users won't know to check the PR queue for this information when upgrading their firmware breaks the serial connection.

photodude committed:
Create README.md with user notes

Added special user note directing users to correct the mks TF28 screen configuration baudrate to match the mainboard configured baudrate.

photodude commented:
@thisiskeithb README.md added with note as requested. Should I also add a link to the MKS TF28 firmware?
I'll make one more commit to the USER instructions README.md to indicate the need to activate babysteps in the screen config as that's also part of the screen config changes

thisiskeithb commented:
> Should I also add a link to the MKS TF28 firmware?

Probably wouldn't hurt.

photodude committed:
Add autoleveling and babysteps screen instructions

MKS TF28 screen configuration extended instructions

- Baud rate
- auto leveling
- Babysteps

photodude committed:
add link to MKS TFT28 screen firmware

- add link to MKS TFT28 screen firmware
- Improve formatting

photodude committed:
formatting improvements

adjust the formatting to improve readability

photodude commented:
@thisiskeithb Have done. This should be complete with the user notes now
